bigquery/bigquery_create_partitioned_biglake_table/main.tf

```diff
@@ -0,0 +1,135 @@
+/**
+* Copyright 2023 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+* This sample creates an Cloud Storage BigLake Table with a partitioning
+* scheme.
+* https://cloud.google.com/bigquery/docs/create-cloud-storage-table-biglake
+*/
+
+# [START bigquery_create_partitioned_biglake_table]
+# Create a bucket where the table is stored.
+# A pre-existing bucket with files maybe used.
+
+# Cloud Storage bucket name must be unique
+resource "random_id" "bucket_name_suffix" {
+  byte_length = 8
+}
+resource "google_storage_bucket" "default" {
+  name                        = "my-bucket-${random_id.bucket_name_suffix.hex}"
+  location                    = "US"
+  force_destroy               = true
+  uniform_bucket_level_access = true
+}
+
+resource "google_storage_bucket_object" "default" {
+  /*
```

Comment:
Two suggestion.
(1) In like 27, you have stated the purpose of before the `resource` definition. You can do the same here.
(2) Here you are using `/*` comments. Not a issues but I suggest simplifying your code comment. Prefer using CGC docs for detailed explanation. As much possible, keep your comment as simple as possible.

Reply:
ack. integrated adriaan's (tech writer) input